graylog2-web-interface/src/views/actions/SearchActions.ts

```diff
@@ -51,6 +53,7 @@ type SearchActionsType = RefluxActions<{
   refresh: () => Promise<void>,
   get: (searchId: SearchId) => Promise<SearchJson>,
   parameters: (parameters: (Array<Parameter> | Immutable.List<Parameter>)) => Promise<View>,
+  setFilterSearchTypes: (FilterSearchTypes) => Promise<void>,
```

Comment:
The parameter needs a name in TypeScript, it should be:
```suggestion
  setFilterSearchTypes: (searchTypes: FilterSearchTypes) => Promise<void>,
```

Without a variable name, it assigns an implicit `any` to the parameter. We can err out later, after we have enabled `noImplicitAny` in the TS config, but this requires some grunt work on our legacy code before.